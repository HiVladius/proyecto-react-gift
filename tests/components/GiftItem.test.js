import React from "react";
import { render } from "@testing-library/react";
import GiftItem from "../../src/components/GifItem";

describe("GiftItem", () => {
  const id = "1";
  const title = "Title";
  const url = "https://example.com/image.jpg";

  test("renders the gift item component", () => {
    const { container } = render(<GiftItem id={id} title={title} url={url} />);
    expect(container).toMatchSnapshot();
  });

  // Add more test cases as needed
});
