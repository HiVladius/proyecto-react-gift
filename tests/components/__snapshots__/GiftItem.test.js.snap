// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GiftItem renders the gift item component 1`] = `
<div>
  <div
    class="card"
  >
    <img
      alt="Title"
      id="1"
      src="https://example.com/image.jpg"
    />
    <p
      id="1"
    >
      Title
    </p>
  </div>
</div>
`;
